### PR TITLE
Willys/oss-goodness

### PR DESCRIPTION
**Pull request by wpschaub**
This PR includes three blog posts to be posted on 4/1, 4/2, and 4/3:

- Azure DevOps Pipeline Blueprints - Open Source Software Sync
- Technology Framework Monitor - Open Source Software Sync 20240322
- DevOps Vancouver Meetup - March 26, 2024

Aim is awarenss of OSS upadtes and the recent meetup.

### CODE REVIEW

File: content/azure-pipeline-blueprints-oss-sync-20240322.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+>
+> [Click here to view the associated Pull Request](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2/pull/27)
+>
+
+Changes:
+
+- **blueprints/nuget-package/azure-pipeline-nuget-package-cd-stage.yml** - eplace the configuration variables with the ```config``` object, which encapsulates an array of data structures..
```
Comment (MerlinDarkstar) [inline content/azure-pipeline-blueprints-oss-sync-20240322.md]: missing the 'r' in 'eplace' (first word)

Comment (MerlinDarkstar) [reply content/azure-pipeline-blueprints-oss-sync-20240322.md]: and extra period at end.

**Review by MerlinDarkstar — DISMISSED** [review]
Great read!

**Review by danielrbroderick — DISMISSED** [review]

File: content/azure-pipeline-blueprints-oss-sync-20240322.md (hunk 1 above)
Comment (wpschaub) [reply content/azure-pipeline-blueprints-oss-sync-20240322.md]: Thank you @MerlinDarkstar. Fixed.

**Review by danielrbroderick — APPROVED** [review]

**Pull request merged**